Contributing.md

```diff
@@ -18,6 +18,18 @@ As an organization contributor, you can merge other people's pull requests, or o
 
 This project is not continuously deployed, there is space for debate after review too. Offering everyone the chance to revert, or make an amending pull request. If it feels right, merge.
 
+#### Bug fixes and Release cycles
+
+We maintain two major branches: `master` and `develop`.
+
+`master` is for working on the current release, so any bug fixes or documentation spelling fixes should be merged into this branch.
```

Comment:
It's such a shame that HTML dictionary lists never became widely used enough to be in the markdown spec.